.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+# Set update schedule for GitHub Actions
+
+version: 2
+updates:
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      # Check for updates to GitHub Actions every week
+      interval: "monthly"
```

Comment:
Comment says `every week` but interval is `monthly`, so need to update one of those. I'm thinking `weekly` would be better but open to opinions.

Reply:
You are absolutely, right, I forgot to change the comment. But now changed it to weekly so it fits.